docs/adr/0002-image-injection-into-remote-clusters-without-native-support.md

```diff
@@ -14,10 +14,8 @@ In order to create any workloads in K8s, an image has to exist on the node or be
 
```

Comment:
rename file to 0003

Reply:
ooop thats super annoying--i had my adr first but it wasnt merged #lame lol